CHANGES.rst

```diff
@@ -1,6 +1,19 @@
 What's new
 ==========
 
+0.6.0 (07-07-2021)
+------------------
+
+New features
+~~~~~~~~~~~~
+- Adaptive masking postprocessing to deal with time-dependent masks (:pull:`29`). By `Stéphane Raynaud <https://github.com/stefraynaud>`_
```

Comment:
```suggestion
- Add the ``skipna`` and ``na_threshold`` options to deal with nD masks (:pull:`29`). By `Stéphane Raynaud <https://github.com/stefraynaud>`_
```
I'm not sure if `nD`is the right term.

Reply:
... to deal with masks that have not just horizontal dimensions.
... to deal with masks that have more than 2 dimensions.
... to deal with 3+ dimensional masks.
... to allow a re-usage of the weights for differently masked data.